api/src/org/apache/cloudstack/api/command/admin/template/CopyTemplateCmdByAdmin.java

```diff
@@ -40,11 +40,20 @@
     @Override
     public void execute() throws ResourceAllocationException{
         try {
+            if (destZoneId == null && (destZoneIds == null || destZoneIds.size() == 0))
+                throw new ServerApiException(ApiErrorCode.PARAM_ERROR,
+                        "Either destzoneid or destzoneids parameters have to be specified.");
+
```

Comment:
Error statement and conditional statement logic are not conveying same message. Some correction is required.